cms/templates/base.html

```diff
@@ -118,5 +118,6 @@
     </script>
     <%include file="widgets/segment-io-footer.html" />
     <div class="modal-cover"></div>
+    <%static:optional_include_mako file="body-post.html" is_theming_enabled="True" />
```

Comment:
Why is this hard-coded, `is_theming_enabled`?

Reply:
This is how optional includes work?

Reply:
https://github.com/edx/edx-platform/blob/b85d6f031c1d6590a184a873fcadb3b2d42024bd/common/djangoapps/pipeline_mako/templates/static_content.html

Reply:
That doesn't answer my question. Why are you hard-coding it, instead of pulling the value from settings or equivalent?

Reply:
Because that's how edx/David Baumgold built this feature and how we've been using it forever.

Reply:
@caesar2164 I don't feel my question is being addressed..

If that's how "Dave Baumgold built this feature", and it's supposed to be hard-coded to `true` then:
- why is there an option to pass in anything other than `true`?
- why does the option default to `false`?
- why are there examples of `optional_include_mako` _not_ passing in a value (meaning they default to `false`)?

Searching the ginkgo branch, there are:
- 7 uses of `optional_include_mako` (excluding tests)
- 2 of them don't pass in `is_theming_enabled`, meaning they default to `false`
- of the 5 uses, at least 2 of them were authored by you

If we exclude our forked changes, that means 40% of upstream occurrences of `optional_include_mako` do _not_ follow that pattern. So that's why I'm confused by a response like "that's how it's done" since there are examples of it _not_ being used that way. Do you see where I'm coming from?

Reply:
@stvstnfrd, from the link I shared earlier:
https://github.com/edx/edx-platform/blob/b85d6f031c1d6590a184a873fcadb3b2d42024bd/common/djangoapps/pipeline_mako/templates/static_content.html#L216

When `is_theming_enabled` set to true causes the platform to check for an optional theme template, which is what we want to do in this case, and what we've meant to do in every case before now (which is why the instances I've authored, and you've approved have it set to true...).

You actually want to hardcode that flag as it's meant as: "THIS IS AN OPTIONAL TEMPLATE THAT IS DEFINED IN THEME".